### ISSUE TITLE
Can not Enable a Disabled Intergration

### ISSUE THREAD

**Issue by afnieves** [home-assistant/frontend#13829]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Can not readable a disabled intergration

### Describe the behavior you expected

Click enable and service is enabled

### Steps to reproduce the issue

1. Settings
2. Devices & Services
3. Click Show Disabled services
4. click enable
...

### What version of Home Assistant Core has the issue?

2022.9.5

### In which browser are you experiencing the issue with?

Google Chrome Version 105.0.5195.125 (Official Build) (arm64)

### Which operating system are you using to run this browser?

macOS Monterey 12.6

**Comment by petslane** [comment]
I can confirm this problem, this affects all integrations. Once the integration is disabled, it can not be enabled or deleted. Noticed this problem quite some time ago, but I do not remember the HA version, but currently, 2022.10.1 still has this problem.

[Untitled_ Oct 27,2022 3_10 PM.webm](https://user-images.githubusercontent.com/2674369/198281211-0d782d17-cfb8-4967-893f-d8dde75a7c54.webm)

**Comment by petslane** [comment]
Tested with fresh HA setup in docker and checked different versions/docker tags:
Working - HA 2022.2.9/FE 20220203.1
Not working - HA 2022.3.0/FE 20220301.0

Seems this bug is added with version 2022.3

**Comment by petslane** [comment]
I found the workaround until it gets fixed.

Unfortunately, I do not have the frontend setup and I'm unable to investigate what is the real problem here, but just poking around I noticed that the integration element `ha-integration-card` has a `disabled` attribute if integration is disabled. Just removing the `disabled` attribute made a 3-dot menu, device and entity links, and enable button work again. So just remove `disabled` attribute to enable disabled integrations. I leave the real fix for someone who knows the framework and how stuff works here.

[Untitled_ Oct 27, 2022 10_00 PM.webm](https://user-images.githubusercontent.com/2674369/198377993-ba6bc875-2976-44d7-9d12-3676a70bd53e.webm)

But poking around a little more it seems to me that `disabled` attribute is something that should not be used. Putting it on any homeassistatn custom html element will make that element and all subelements to be disabled, eg. no clicks are registered on menus, links, buttons etc. This `disabled` attribute was [added about 2 years ago](https://github.com/home-assistant/frontend/pull/8442) but enabling functionality broke with version 2022.3, so something else changed that handled this attribute differently.
